server/src/main/java/io/deephaven/server/console/completer/PythonAutoCompleteObserver.java

```diff
@@ -101,33 +127,18 @@ private void getCompletionItems(GetCompletionItemsRequest request,
             StreamObserver<AutoCompleteResponse> responseObserver) {
         final ScriptSession scriptSession = exportedConsole.get();
         try (final SafeCloseable ignored = scriptSession.getExecutionContext().open()) {
-
-            PyObject completer = (PyObject) scriptSession.getVariable("jedi_settings");
-            boolean canJedi = completer.callMethod("is_enabled").getBooleanValue();
-            if (!canJedi) {
-                log.trace().append("Ignoring completion request because jedi is disabled").endl();
-                // send back an empty, failed response...
-                safelyExecuteLocked(responseObserver,
-                        () -> responseObserver.onNext(AutoCompleteResponse.newBuilder()
-                                .setCompletionItems(GetCompletionItemsResponse.newBuilder()
-                                        .setSuccess(false)
-                                        .setRequestId(request.getRequestId()))
-                                .build()));
-                return;
-            }
```

Comment:
what happens if client uses `jedi_settings.mode = 'OFF'` to disable autocomplete in their session?